storage/watchable_store.go

```diff
@@ -320,12 +330,18 @@ func (s *watchableStore) notify(rev int64, evs []storagepb.Event) {
 		if eb.revs != 1 {
 			panic("unexpected multiple revisions in notification")
 		}
+		w.mu.Lock()
```

Comment:
i guess i was wrong. but where do we actually get w.cur updated when the watcher move forward with revision?